main.go

```diff
@@ -1,7 +1,60 @@
 package main
 
-import "github.com/odpf/compass/cli"
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/odpf/compass/cli"
+)
+
+const (
+	exitOK    = 0
+	exitError = 1
+)
 
 func main() {
-	cli.Execute()
+	cliConfig, err := cli.LoadConfig()
+	if err != nil {
+		cliConfig = &cli.Config{}
+	}
+
+	rootCmd := cli.New(cliConfig)
+
+	if cmd, err := rootCmd.ExecuteC(); err != nil {
```

Comment:
Can we run with `ExecuteContextC` instead? We can pass in a context created with `signal.NotifyContext`.